examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -515,6 +515,22 @@ def parse_args(input_args=None):
         type=int,
         help="Print the loss for every logging_step.",
     )
+    parser.add_argument(
+        "--mediapipe",
+        default="",
+        type=str,
+        help="Use gaudi2 HW mediapipe over regular dataloader. \
+        case 1: nothing is passed to this argument -> regular torch dataloader is used\
+        case 2: an empty or non existant path is passed -> images are dumped from dataset (passed in through dataset_name) in that location before first run \
+        case 3: a non empty path is passed -> images from that location are used ",
+    )
+    parser.add_argument(
+        "--discount_chkpoint_saving_in_throughput",
+        default=False,
+        action="store_true",
+        help="Checkpoitn saving takes a lot of time. Ignore time for checkpoint saving for throughput calculations"
```

Comment:
can you align with https://github.com/huggingface/optimum-habana/blob/20832c92ce3e9cfe94aeb1797a5533b4be962a6c/optimum/habana/transformers/training_args.py#L187 ?